modular_zubbers/code/datums/emotes.dm

```diff
@@ -0,0 +1,180 @@
+/datum/emote/living/tilt
+	key = "tilt"
+	key_third_person = "tilts"
+	message = "tilts their head."
+	emote_type = EMOTE_VISIBLE
+
+/datum/emote/living/squint
+	key = "squint"
+	key_third_person = "squints"
+	message = "squints their eyes." // i dumb
+	emote_type = EMOTE_VISIBLE
+
+/datum/emote/living/mlem
+	key = "mlem"
+	key_third_person = "mlems"
+	message = "sticks their tongue for a moment. Mlem!"
+	emote_type = EMOTE_VISIBLE
+
+/datum/emote/living/audio/cackle
+	key = "cackle"
+	key_third_person = "cackles"
+	message = "cackles hysterically!"
+	message_mime = "cackles silently!"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/cackle_yeen.ogg'
+
+/datum/emote/living/audio/caw
+	key = "caw"
+	key_third_person = "caws"
+	message = "caws!"
+	message_mime = "caws silently!"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/caw.ogg'
+
+/datum/emote/living/audio/bleat
+	key = "bleat"
+	key_third_person = "bleats"
+	message = "bleats loudly!"
+	message_mime = "bleats silently!"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/bleat.ogg'
+
+/datum/emote/living/audio/chitter2
+	key = "chitter2"
+	key_third_person = "chitters2"
+	message = "chitters."
+	message_mime = "chitters silently!"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/moth/mothchitter2.ogg'
+
+/datum/emote/living/audio/boowomp
+	key = "boowomp"
+	key_third_person = "boowomps"
+	message = "produces a sad boowomp."
+	message_mime = "produces a silent boowomp."
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/boowomp.ogg'
```

Comment:
LRP

Reply:
aww, I loved boowomp :( can get rid of it though